Riot/Modules/Spaces/SpaceRoomList/ExploreRoomCoordinator.swift

```diff
@@ -28,6 +28,7 @@ final class ExploreRoomCoordinator: ExploreRoomCoordinatorType {
     private let navigationRouter: NavigationRouterType
     private let session: MXSession
     private let spaceId: String
+    private var spaceIdStack: [String]
```

Comment:
Could this one have a doc comment to explain what it is? I can see from below but at first glance I had to go looking.